Fixes "make install" errors

  /usr/lib/qt5/bin/qmake -install qinstall /builddir/gsettings-qt-d5e002d7e0bce46c315bcc99a44a8bd51f49f488/GSettings/qmldir /destdir/aarch64-linux-musl/gsettings-qt-0.2r211/usr/aarch64-linux-musl/usr/lib/qt5/qml/GSettings.1.0/qmldir
  export LD_LIBRARY_PATH=/builddir/gsettings-qt-d5e002d7e0bce46c315bcc99a44a8bd51f49f488/GSettings/../src; /usr/lib/qt5/bin/qmlplugindump -notrelocatable GSettings 1.0 .. > /destdir/aarch64-linux-musl/gsettings-qt-0.2r211//usr/aarch64-linux-musl/usr/lib/qt5/qml/GSettings.1.0/plugins.qmltypes
  /bin/sh: line 1: /usr/lib/qt5/bin/qmlplugindump: No such file or directory
  make[1]: *** [Makefile.gsettings-qt:1905: install_qmltypes] Error 127

Adding qt5-declarative to hostmakedepends results in

  QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-...'
  QQmlComponent: Component is not ready
  make[1]: *** [Makefile.gsettings-qt:1905: install_qmltypes] Error 3

-> are we just missing runtime qml plugin deps for target?

FIXME: UPSTREAM THIS!
---
diff --git a/GSettings/gsettings-qt.pro b/GSettings/gsettings-qt.pro
index 2642981..2c87247 100644
--- a/GSettings/gsettings-qt.pro
+++ b/GSettings/gsettings-qt.pro
@@ -23,8 +23,9 @@ extra.path = $$installPath
 extra.files += qmldir
 INSTALLS += extra
 
-qmltypes.path = $$installPath
-qmltypes.files = plugins.qmltypes
-qmltypes.extra = export LD_LIBRARY_PATH=$$PWD/../src; $$[QT_INSTALL_BINS]/qmlplugindump -notrelocatable GSettings 1.0 .. > $(INSTALL_ROOT)/$$installPath/plugins.qmltypes
-INSTALLS += qmltypes
-
+!cross_compile {
+	qmltypes.path = $$installPath
+	qmltypes.files = plugins.qmltypes
+	qmltypes.extra = export LD_LIBRARY_PATH=$$PWD/../src; $$[QT_INSTALL_BINS]/qmlplugindump -notrelocatable GSettings 1.0 .. > $(INSTALL_ROOT)/$$installPath/plugins.qmltypes
+	INSTALLS += qmltypes
+}
